src/tictactoe/ui.clj

```diff
@@ -0,0 +1,14 @@
+(ns tictactoe.ui
+  (:require [tictactoe.board :as rb])
```

Comment:
Same for `rb`. I like aliases to shorten the full name, but I think they should still be descriptive